src/screens/admin/AdminDashboard.tsx

```
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+	Box,
+	VStack,
+	Text,
+	Tabs,
+	TabList,
+	TabPanels,
+	Tab,
+	TabPanel,
+	Alert,
```

ESLint

[error] 'Alert' is defined but never used.

(@typescript-eslint/no-unused-vars)